SpywareFilter/sections/specific.txt

```diff
@@ -53,6 +53,7 @@
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp|imore.com|city.kushiro.lg.jp|emeraldbridal.com.au|emarat-news.ae|imobie.jp
 ||assets.adobedtm.com^$domain=mcclatchydc.com|elnuevoherald.com|bellinghamherald.com|sacbee.com|charlotteobserver.com|newsobserver.com|miamiherald.com|ef.ru|algida.com.tr|bmw-motorrad.co.uk|cif.com.tr|clearhaircare.com|domestos.com|dove.com|elidor.com.tr|f1sport.autorevue.cz|geico.com|hellmanns.com.tr|herkessofraya.com|johnsonsbaby.com.tr|lipton.com|lnk.to|lux.com|magnumicecream.com|neutrogena.com.tr|omo.com|rexona.com|samsung.com|sap.com|schwarzkopf.com.tr|signalturkiye.com|msnbc.com|nbcnews.com|kbb.com|eurosport.ru|timgate.it|intermiles.com|oxfordmail.co.uk|spiegel.de|hp.com|deutschepost.de|thebay.com
 !
+||pixel.archive.*^$domain=~archive.org
```

Comment:
Why archive.org is excluded?
Do they have many mirrors?

Reply:
Please read the attached Wikipedia article.

Reply:
But your rule cannot break archive.org 
I don't understand.

Reply:
Because the owner is different, I excluded archive.org.
Do you want to include the website too?

Reply:
`$domain=~archive.org` just looks like a fix for some problem, but there is no problem) It's better to remove it so there's no confusion.

Reply:
Done.